controllers/defaults.go

```diff
@@ -28,6 +28,7 @@ var (
 		"CSI_PROVISIONER_IMAGE":   "k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0",
 		"CSI_LIVENESSPROBE_IMAGE": "k8s.gcr.io/sig-storage/livenessprobe:v2.5.0",
 		"CSI_RESIZER_IMAGE":       "k8s.gcr.io/sig-storage/csi-resizer:v1.3.0",
+		"CSI_SNAPSHOTER_IMAGE":    "k8s.gcr.io/sig-storage/snapshot-controller:v4.2.1",
```

Comment:
This is wrong, you should not use snapshot-controller you should use external-snapshotter https://kubernetes-csi.github.io/docs/external-snapshotter.html

Reply:
Thanks for pointing it out changed it.